airflow/dag_processing/bundles/git.py

```diff
@@ -60,6 +61,7 @@ def get_ui_field_behaviour(cls) -> dict[str, Any]:
                 "extra": json.dumps(
                     {
                         "key_file": "optional/path/to/keyfile",
+                        "private_key": "optional/private_key",
```

Comment:
nit: maybe say "optional inline private key", just don't make it look like a path basically.

Reply:
Great, Thanks.